pandas/tests/test_window.py

```diff
@@ -520,6 +520,39 @@ def test_iter_raises(self, klass):
         with pytest.raises(NotImplementedError):
             iter(obj.rolling(2))
 
+    def test_agg(self):
+        # GH 15072
+        df = pd.DataFrame([
+            ['A', 10, 20],
+            ['A', 20, 30],
+            ['A', 30, 40],
+            ['B', 10, 30],
+            ['B', 30, 40],
+            ['B', 40, 80],
+            ['B', 80, 90]], columns=['stock', 'low', 'high'])
+
+        index = pd.MultiIndex.from_tuples([
+            ('A', 0), ('A', 1), ('A', 2),
+            ('B', 3), ('B', 4), ('B', 5), ('B', 6)], names=['stock', None])
+        columns = pd.MultiIndex.from_tuples([
+            ('low', 'mean'), ('low', 'max'), ('high', 'mean'),
+            ('high', 'min')])
+        expected = pd.DataFrame([
+            [np.nan, np.nan, np.nan, np.nan],
```

Comment:
maye put in TestApi. this should test rolling & expanding (atm we don't have a nice modular structure to do this :<)